com.playeveryware.eos/Runtime/Core/EOS_SDK_Additions/EOSClientCredentials.cs

```diff
@@ -29,13 +29,21 @@ namespace PlayEveryWare.EpicOnlineServices
     using PlayEveryWare.EpicOnlineServices.Utility;
     using System;
     using System.Security.Cryptography;
+    using System.Text.RegularExpressions;
 
     public class EOSClientCredentials : IEquatable<EOSClientCredentials>
     {
         public string ClientId;
         public string ClientSecret;
         public readonly string EncryptionKey;
 
+        private static readonly Regex s_invalidEncryptionKeyRegex;
+
+        static EOSClientCredentials()
+        {
+            s_invalidEncryptionKeyRegex = new Regex("[^0-9a-fA-F]");
```

Comment:
I didn't know the encryption key had to be a GUID. Is this a new limitation, or something that was always secretly present?

Reply:
> This 64-character hexadecimal string builds a 256-bit key that EOS uses to encrypt user data.
https://dev.epicgames.com/docs/game-services/title-storage

This is entirely correct, it should be a hex of this length. I wonder if there's anyone out there who is in violation of this rule that'll be hosed by the update, or if was never valid to provide a key that didn't match these criteria.

Reply:
Do you happen to know if this field should be case sensitive?

Reply:
I believe that the field is case sensitive, in that an encryption key with different case cannot be used to decrypt a file encrypted with the original case.